zc_install/includes/nginx_conf/ngx_conf_server.txt

```diff
@@ -0,0 +1,320 @@
+	location ~ %%store_folder%%/media {
+		return 403;
+	}
+
+
+	#############################################################################################
+	# ZEN CART 'extras' Folder
+	#############################################################################################	
+	## Allow all requests to html files in 'extras'
+	location ~ %%store_folder%%/extras/(.+)\.html$ {
+		expires $zencart_expires;		
+	}
+	## Allow all requests to all PHP files
+	location ~ %%store_folder%%/extras/(.+)\.php$ {
+		error_page 418 = @zencart_php_handler;
+		return 418;
+	}
+	## Deny any other request to 'extras' and any subfolders
+	location ~ %%store_folder%%/extras {
+		return 403;
+	}
+
+
+	#############################################################################################
+	# ZEN CART 'email' Folder
+	#############################################################################################	
+	## Allow all requests to specific static file types in 'email'
+	location ~ %%store_folder%%/email/(.+)\.(jpe?g|gif|png)$ {
+		expires $zencart_expires;		
+	}
+	## Deny any other request to 'email' and any subfolders
+	location ~ %%store_folder%%/email {
+		return 403;
+	}
+
+
+	#############################################################################################
+	# ZEN CART 'editors' and 'docs' Folders
+	#############################################################################################	
+	## Allow all requests to specific static file types in 'editors' or 'docs'
+	location ~ %%store_folder%%/(editors|docs)/(.+)\.(js|css|jpe?g|gif|png|html?|xml|pdf)$ {
+		expires $zencart_expires;		
+	}
+	## Deny any other request to 'editors' or 'docs' and any subfolders
+	location ~ %%store_folder%%/(editors|docs) {
+		return 403;
+	}
+
+
+	#############################################################################################
+	# ZEN CART 'cache' Folder
+	#############################################################################################	
+	## Allow and fastrack internal requests to 'cache'
+	location ^~ %%store_folder%%/cache {
+		internal;
+	}
+
+
+	#############################################################################################
+	# ZEN CART 'logs' Folder
+	#############################################################################################	
+	## Allow and fastrack internal requests to 'logs'
+	location ^~ %%store_folder%%/logs {
+		internal;
+	}
+
+
+	#############################################################################################
+	# ZEN CART ADMIN Folder
+	#############################################################################################	
+	## Allow all 'POST', 'GET' and 'HEAD' requests to all PHP files
+	location ~ %%admin_folder%%/(.+)\.php$ {
+		if ( $request_method !~ ^(GET|HEAD|POST)$ ) { return 403; }
+		error_page 418 = @zencart_php_handler;
+		return 418;
+	}
+	## Allow all 'GET' and 'HEAD' requests to specific static file types
+	location ~ %%admin_folder%%(.*)$ {
```

Comment:
Should line 272 have a `/` before the `(.*)` ... like the other directives in this section do?